client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -660,9 +519,8 @@ class AdsFormSettings extends Component {
 
 						{ this.props.siteIsJetpack ? this.adstxt() : null }
 
-						<FormSectionHeading>{ translate( 'Site Owner Information' ) }</FormSectionHeading>
-						{ this.siteOwnerOptions() }
-						{ this.state.us_checked ? this.taxOptions() : null }
+						<FormSectionHeading>{ translate( 'Payment Information' ) }</FormSectionHeading>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Stored Payment Information' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-wordads-owner-data)